library/node

```diff
@@ -3,114 +3,114 @@
 Maintainers: The Node.js Docker Team <https://github.com/nodejs/docker-node> (@nodejs)
 GitRepo: https://github.com/nodejs/docker-node.git
 
-Tags: 13.7.0-stretch, 13.7-stretch, 13-stretch, stretch, 13.7.0, 13.7, 13, latest
+Tags: 13.8.0-stretch, 13.8-stretch, 13-stretch, stretch, 13.8.0, 13.8, 13, latest
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 975ad7f092d362ade47f4cff6e2897de17be4cc5
+GitCommit: 46f5203674c748b0701135c8eeea4b250b3ecb6d
 Directory: 13/stretch
 
-Tags: 13.7.0-stretch-slim, 13.7-stretch-slim, 13-stretch-slim, stretch-slim, 13.7.0-slim, 13.7-slim, 13-slim, slim
+Tags: 13.8.0-stretch-slim, 13.8-stretch-slim, 13-stretch-slim, stretch-slim, 13.8.0-slim, 13.8-slim, 13-slim, slim
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 975ad7f092d362ade47f4cff6e2897de17be4cc5
+GitCommit: 46f5203674c748b0701135c8eeea4b250b3ecb6d
 Directory: 13/stretch-slim
 
-Tags: 13.7.0-buster, 13.7-buster, 13-buster, buster
+Tags: 13.8.0-buster, 13.8-buster, 13-buster, buster
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 975ad7f092d362ade47f4cff6e2897de17be4cc5
+GitCommit: 46f5203674c748b0701135c8eeea4b250b3ecb6d
 Directory: 13/buster
 
-Tags: 13.7.0-buster-slim, 13.7-buster-slim, 13-buster-slim, buster-slim
+Tags: 13.8.0-buster-slim, 13.8-buster-slim, 13-buster-slim, buster-slim
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 975ad7f092d362ade47f4cff6e2897de17be4cc5
+GitCommit: 46f5203674c748b0701135c8eeea4b250b3ecb6d
 Directory: 13/buster-slim
 
-Tags: 13.7.0-alpine3.10, 13.7-alpine3.10, 13-alpine3.10, alpine3.10
+Tags: 13.8.0-alpine3.10, 13.8-alpine3.10, 13-alpine3.10, alpine3.10
```

Comment:
The Alpine tags for 13 shouldn't be changing since it was updated due to other issues

Reply:
that's true... bug in the script (probably due to other v13s being updated)

Reply:
I'm just about to call it a night - if you could open up a manual PR without these changes that'd be great 👍

Reply:
GitHub wouldn't let me apply the suggestions directly through the web interface for some reason, but I was able to push them to this PR reverting the Node.js 13 Alpine-related tags back to 13.7.0.  I'll get a build test going right away.